cmd/minishift/cmd/addon/addons_list.go

```diff
@@ -88,14 +89,15 @@ func runListCommand(cmd *cobra.Command, args []string) {
 func printAddOnList(manager *manager.AddOnManager, writer io.Writer, template *template.Template) {
 	addOns := manager.List()
 	sort.Sort(addon.ByStatusThenPriorityThenName(addOns))
+	display := new(tabwriter.Writer)
+	display.Init(os.Stdout, 0, 8, 0, '\t', 0)
+
 	for _, addon := range addOns {
 		description := strings.Join(addon.MetaData().Description(), fmt.Sprintf("\n%13s", " "))
```

Comment:
I don't see the description printed in the o/p, and also what is the difference between `fmt.Sprintf("\n%13s", " ")` vs. `fmt.Sprintf("\n%s", " ")` ?